src/System.Device.Gpio/System/Device/Spi/Drivers/Windows10SpiDevice.Windows.cs

```diff
@@ -23,6 +24,11 @@ public class Windows10SpiDevice : SpiDevice
         /// </param>
         public Windows10SpiDevice(SpiConnectionSettings settings)
         {
+            if (settings.DataFlow != DataFlow.MsbFirst || settings.ChipSelectLineActiveState != PinValue.Low)
+            {
+                throw new PlatformNotSupportedException($"Changing {nameof(settings.DataFlow)} or {nameof(settings.ChipSelectLineActiveState)} options is not supported on the current platform.");

```

Comment:
I had a question related to this and when a platform doesn't support something another does.  Is this a feature the team might ping the Windows team to possibly add in a future release (and WinRT APIs)?  Just curious.

Reply:
We can send request to Windows team but we cannot force or promise they will do it. This seems like a simple API addition though and useful so I suspect we might be able to get it. I think in order to avoid spamming them with various requests we should create one issue which summarizes them all and I can send that forward to them.

Reply:
Should we create an issue to log some of these for future review?  A couple of others that come to mind are possible CAN and I2S.